.github/workflows/tests.yml

```diff
@@ -65,8 +67,14 @@ jobs:
         id: install-tardis
         # shell: bash -l {0}
         run: |
-          pip install git+https://github.com/tardis-sn/tardis.git
-          
+            if [ "${{ github.event.action }}" == "Tardis-Dispatch" ]; then
+              echo "Triggered by repository dispatch, installing the latest version of TARDIS."
+              pip install git+https://github.com/${{github.repository_owner}}/tardis.git@${{ github.event.client_payload.commit_sha }}
+            else
+              echo "Not triggered by repository dispatch, installing the latest version of TARDIS."
+              pip install git+https://github.com/${{ github.repository_owner }}/tardis.git
```

Comment:
repo owner will always be tardis-sn right?

Reply:
I used repository_owner, so that it is easy to test on forks, otherwise anyone who tests this will have to rename tardis-sn to their personal fork everytime.